source/mir/cv/core/simd.d

```diff
@@ -0,0 +1,77 @@
+/+
+    Utilities to help use SIMD vectors.
++/
+
+module mir.cv.core.simd;
```

Comment:
I would propose another approach that is used in mir-glas, see [glas/internal/config](https://github.com/libmir/mir-glas/blob/master/source/glas/internal/config.d).

The reason is `mir-cv` is open source and boost licensed and d compilers are fast. LDC supports compile time traits for CPU information (see glas internal config). Lets move forward with LDC for betterC libraries. Front-end DCV users would be able to use DMD because extern(C) interface.